models/demos/t3000/llama2_70b/tests/test_llama_model.py

```diff
@@ -219,33 +298,34 @@ def run_test_LlamaModel_inference(
     logger.info(f"Average Top-5 over {len(all_top5)} tokens: {sum(all_top5) / len(all_top5)}")
     # Check kv cache
     # PyTorch output --------------------------------------------------------------------
-    pytorch_layer_present = [
-        pytorch_model.model.layers[0]
-        .attention.cache_k.clone()
-        .permute(0, 2, 1, 3)[:batch, ...],  # [batch, n_kv_heads, seq, head_dim]
-        pytorch_model.model.layers[0]
-        .attention.cache_v.clone()
-        .permute(0, 2, 1, 3)[:batch, ...],  # [batch, n_kv_heads, seq, head_dim]
-    ]
-
-    tt_layer_present_all = [ttnn.from_device(lp) for lp in tt_model.layers[0].attention.layer_past]
-    tt_layer_present_all = [
-        ttnn.to_torch(lp, mesh_composer=ConcatMeshToTensor(t3k_mesh_device, dim=1))[:batch, ...]
-        for lp in tt_layer_present_all
-    ]
-
-    cache_test_pass = check_kv_cache(
-        pytorch_layer_present,
-        tt_layer_present_all,
-        generation_start_pos,
-        generation_length,
-        seq_len,
-        mode == "prefill",
-        pcc,
-    )
-    all_tests_pass = all_tests_pass and cache_test_pass
-    if all_tests_pass:
-        logger.info(f"{llama_version} output Passed!")
+    if chunk_size is None:
```

Comment:
Should cache_test_pass also be checked for chunked prefill by unpaging the caches?